cb-sol-cli/cmd/bridge.js

```diff
@@ -113,6 +113,112 @@ const queryResourceId = new Command("query-resource")
+
+        log(args,`Setting contract ${args.tokenContract} as burnable on handler ${args.handler}`);
+        const tx = await bridgeInstance.adminSetBurnable(args.handler, args.tokenContract, { gasPrice: constants.OPTIMISM_GAS_PRICE, gasLimit: constants.OPTIMISM_GAS_LIMIT});
+        await waitForTx(args.provider, tx.hash)
+    })
+
+const queryIsBurnCmdOptimism = new Command("query-is-burn-optimism")
+    .description("Get whether the token address is registered as mintable/burnable in a handler")
+    .option('--handler <address>', 'Handler contract address', constants.ERC20_HANDLER_ADDRESS)
+    .option('--tokenContract <address>', 'Token contract being queried if it is burnable', constants.ERC20_ADDRESS)
+    .action(async function (args) {
+        await setupParentArgs(args, args.parent.parent)
+
+        const handlerInstance = new ethers.Contract(args.handler, constants.ContractABIsOptimism.HandlerHelpers.abi, args.wallet)
+        const isBurn = await handlerInstance._burnList(args.tokenContract)
+        log(args, `Getting if contract ${args.tokenContract} is registered as burnable on handler ${args.handler}: ${isBurn}`)
+    })
+
+const cancelProposalCmdOptimism = new Command("cancel-proposal-optimism")
+    .description("Cancel a proposal that has passed the expiry threshold")
+    .option('--bridge <address>', 'Bridge contract address', constants.BRIDGE_ADDRESS)
+    .option('--chainId <id>', 'Chain ID of proposal to cancel', 0)
+    .option('--depositNonce <value>', 'Deposit nonce of proposal to cancel', 0)
+    .action(async function (args) {
+        await setupParentArgs(args, args.parent.parent)
+        const bridgeInstance = new ethers.Contract(args.bridge, constants.ContractABIsOptimism.Bridge.abi, args.wallet);
+
+        log(args, `Setting proposal with chain ID ${args.chainId} and deposit nonce ${args.depositNonce} status to 'Cancelled`);
+        const tx = await bridgeInstance.adminCancelProposal(args.chainId, args.depositNonce);
+        await waitForTx(args.provider, tx.hash)
+    })
+
+const queryProposalCmdOptimism = new Command("query-proposal-optimism")
+    .description("Queries a proposal")
+    .option('--bridge <address>', 'Bridge contract address', constants.BRIDGE_ADDRESS)
+    .option('--chainId <id>', 'Source chain ID of proposal', 0)
+    .option('--depositNonce <value>', 'Deposit nonce of proposal', 0)
+    .option('--dataHash <value>', 'Hash of proposal metadata', constants.ERC20_PROPOSAL_HASH)
+    .action(async function (args) {
+        await setupParentArgs(args, args.parent.parent)
+        const bridgeInstance = new ethers.Contract(args.bridge, constants.ContractABIsOptimism.Bridge.abi, args.wallet);
+
+        const prop = await bridgeInstance.getProposal(args.chainId, args.depositNonce, args.dataHash)
+
+        console.log(prop)
+    })
+
+const queryResourceIdOptimism = new Command("query-resource-optimism")
```

Comment:
Semantics..

Function name `queryResourceIdOptimism` includes `Id` and does not include `Cmd` whereas other commands also relating to resource ID does not include `Id` and do include `Cmd`

`queryResourceIdOptimism` => `queryResourceCmdOptimism`